formicaio/docker-compose.yml

```
@@ -0,0 +1,29 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: formicaio_formicaio_proxy_1
+      APP_PORT: 52100
+    container_name: formicaio-app_proxy_1
```

container_name is not needed.
```suggestion
      APP_PORT: 52100

```